### PR TITLE
add orjson for jsonresponse

### PR DESCRIPTION
**Pull request by michaelfeil**
## Motivation

ORJson allows for faster JSON serialization of responses. 
This could reduce latency and help reducing the cpu overhead.

I heavily use it for in infinity, e.g. https://github.com/michaelfeil/infinity/blob/4e380e110b72e61a159108dabfbd1fa27506883c/libs/infinity_emb/infinity_emb/infinity_server.py#L251

## Checklist

- [x] Format your code according to the [Contributor Guide](https://github.com/sgl-project/sglang/blob/main/docs/en/contributor_guide.md).
I tried to format but hit this error.
```
$ pre-commit run --all-files
[INFO] Once installed this environment will be reused.
[INFO] This may take a few minutes...
An unexpected error has occurred: CalledProcessError: command: ('/home/michael/miniconda3/bin/python', '-mvirtualenv', '/home/michael/.cache/pre-commit/repo4nmu5puv/py_env-python3.9', '-p', 'python3.9')
return code: 1
stdout:
    RuntimeError: failed to find interpreter for Builtin discover of python_spec='python3.9'
stderr: (none)
```

- [ ] Add unit tests as outlined in the [Contributor Guide](https://github.com/sgl-project/sglang/blob/main/docs/en/contributor_guide.md).
- [x] Update documentation as needed, including docstrings or example tutorials.

### CODE REVIEW

**Comment by michaelfeil** [comment]
@liangan1 re: sglang webinar

File: python/pyproject.toml [hunk 1]
```diff
@@ -21,6 +21,7 @@ dependencies = [
 
 [project.optional-dependencies]
 runtime_common = ["aiohttp", "decord", "fastapi", "hf_transfer", "huggingface_hub", "interegular",
+    "orjson",
     "packaging", "pillow", "psutil", "pydantic", "python-multipart",
```
Comment (merrymercy) [inline python/pyproject.toml]:
```suggestion
    "orjson", "packaging", "pillow", "psutil", "pydantic", "python-multipart",
```

**Review by merrymercy — APPROVED** [review]

**Comment by merrymercy** [comment]
@michaelfeil Thanks for the contribution. This looks good and we can merge it.

**Comment by zhyncs** [comment]
@michaelfeil So fast LOL

**Comment by Ying1123** [comment]
As fast as an AI bot ୧(๑•̀◡•́๑)૭

**Pull request merged**